### PR TITLE
Enhance AutoYast Configuration

### PR DESCRIPTION
**Pull request by RoyCai7**
Main change:

Networking Configuration (interfaces Section):
Removed interface configurations in the networking setup, ensuring proper settings for network interfaces.

Partitioning Configuration (partitioning Section): Optimized the disk device selection by using /dev/disk/by-id/ with a device ID, providing flexibility for different hardware. In the partition section, removed an unnecessary partition block, making the configuration more concise.

Other changes.
IOMMU setting based on CPU type.

progress ticket: https://progress.opensuse.org/issues/152747

- Related ticket: https://progress.opensuse.org/issues/152747
- Needles: none
- Verification run:

- [x] [Buildsles15sp5:xen:wizard](http://10.100.103.119/tests/overview?distri=sle&version=15-SP5&build=sles15sp5%3Axen%3Awizard&groupid=3)
- [x] [Buildsles15sp5:xen:lipa](http://10.100.103.119/tests/overview?distri=sle&version=15-SP5&build=sles15sp5%3Axen%3Alipa&groupid=3)
- [x] [Buildsles15sp5:xen:inara](http://10.100.103.119/tests/overview?distri=sle&version=15-SP5&build=sles15sp5%3Axen%3Ainara&groupid=3)
- [x] [Buildsles15sp5:xen:briza](http://10.100.103.119/tests/overview?distri=sle&version=15-SP5&build=sles15sp5%3Axen%3Abriza&groupid=3)
- [x] [Buildsles15sp5:kvm:wizard](http://10.100.103.119/tests/overview?distri=sle&version=15-SP5&build=sles15sp5%3Akvm%3Awizard&groupid=3)
- [x] [Buildsles15sp5:kvm:lipa](http://10.100.103.119/tests/overview?distri=sle&version=15-SP5&build=sles15sp5%3Akvm%3Alipa&groupid=3)
- [x] [Buildsles15sp5:kvm:inara](http://10.100.103.119/tests/overview?distri=sle&version=15-SP5&build=sles15sp5%3Akvm%3Ainara&groupid=3)
- [x] [Buildsles15sp5:kvm:briza](http://10.100.103.119/tests/overview?distri=sle&version=15-SP5&build=sles15sp5%3Akvm%3Abriza&groupid=3)
- [x] [Buildsles12sp5:xen:wizard](http://10.100.103.119/tests/overview?distri=sle&version=12-SP5&build=sles12sp5%3Axen%3Awizard&groupid=3)
- [x] [Buildsles12sp5:xen:lipa](http://10.100.103.119/tests/overview?distri=sle&version=12-SP5&build=sles12sp5%3Axen%3Alipa&groupid=3)
- [x] [Buildsles12sp5:xen:inara](http://10.100.103.119/tests/overview?distri=sle&version=12-SP5&build=sles12sp5%3Axen%3Ainara&groupid=3)
- [x] [Buildsles12sp5:xen:briza](http://10.100.103.119/tests/overview?distri=sle&version=12-SP5&build=sles12sp5%3Axen%3Abriza&groupid=3)
- [x] [Buildsles12sp5:kvm:wizard](http://10.100.103.119/tests/overview?distri=sle&version=12-SP5&build=sles12sp5%3Akvm%3Awizard&groupid=3)
- [x] [Buildsles12sp5:kvm:lipa](http://10.100.103.119/tests/overview?distri=sle&version=12-SP5&build=sles12sp5%3Akvm%3Alipa&groupid=3)
- [x] [Buildsles12sp5:kvm:inara](http://10.100.103.119/tests/overview?distri=sle&version=12-SP5&build=sles12sp5%3Akvm%3Ainara&groupid=3)
- [x] [Buildsles12sp5:kvm:briza](http://10.100.103.119/tests/overview?distri=sle&version=12-SP5&build=sles12sp5%3Akvm%3Abriza&groupid=3)

### CODE REVIEW

File: data/virtualization/autoyast/host_12.xml.ep [hunk 1]
```diff
@@ -119,24 +119,15 @@
...
   <partitioning config:type="list">
     <drive>
-      % if ($check_var->('IPMI_HW', 'supermicro')) {
-      <device>/dev/sda</device>
-      % }
-      <disklabel>msdos</disklabel>
+  % my $wwn = {'lipa' => 'wwn-0x5002538c4002f8bd', briza => 'wwn-0x5000c5004f0e566d'};
+  % my $hostname = (split(/\./, $get_var->("SUT_IP")))[0];
+  % my $device_id = defined($wwn->{$hostname}) ? '/dev/disk/by-id/' . $wwn->{$hostname} : '';
+      <device><%= $device_id %></device>
```
Comment (alice-suse) [inline data/virtualization/autoyast/host_12.xml.ep:129]: For SUTs other than lipa or briza, device_id will be empty. Is it okay?

Comment (RoyCai7) [reply data/virtualization/autoyast/host_12.xml.ep:129]: if mutil disks, it display '<device>/dev/disk/by-id/wwn-0x5002538c4002f8bd</device>'
if single disk, it display '<device></device>', the system will be installed on the available disk randomly. 
I think it is logical.

Comment (alice-suse) [reply data/virtualization/autoyast/host_12.xml.ep:129]: ok

**Review by alice-suse — APPROVED** [review]
LGTM

**Comment by Julie-CAO** [comment]
LGTM

**Pull request merged**